server/repository/src/migrations/v2_06_01/change_vaccination_date_to_nullable.rs

```diff
@@ -11,8 +11,12 @@ impl MigrationFragment for Migrate {
         if cfg!(feature = "postgres") {
             sql!(
                 connection,
+                // Also drop FK reference that we removed for sqlite in 2.5 but not for postgres
+                // name_link may not exist everywhere that vaccination is synced, so no reference constraint
```

Comment:
Maybe this is cheeky to sneak in here... but should have been in 2.5, so figured sooner the better?